apps/cowswap-frontend/src/modules/tradeQuote/hooks/useTradeQuotePolling.ts

```diff
@@ -115,8 +121,7 @@ export function useTradeQuotePolling() {
         /**
          * Reset the quote state in order to not trigger the quote expiration check again
          */
-        tradeQuoteManager.reset()
-        fetchAndUpdateQuote(false)
+        fetchAndUpdateQuote(false, true)
```

Comment:
This is the most important change.
When quote get expired, it was resetting current quote, but after it might skip the quote updating, so we got stuck in the state without current quote.

Reply:
Now it doesn't reset current quote, but forcely update it